README.md

```diff
@@ -8,8 +8,7 @@ A Swift mixin to use `UITableViewCells`, `UICollectionViewCells` and `UIViewCont
 
 [![Platform](http://cocoapod-badges.herokuapp.com/p/Reusable/badge.png)](http://cocoadocs.org/docsets/Reusable)
 [![Version](http://cocoapod-badges.herokuapp.com/v/Reusable/badge.png)](http://cocoadocs.org/docsets/Reusable)
-![Swift 2.2](https://img.shields.io/badge/Swift-2.2-orange.svg)
-![Swift 2.3](https://img.shields.io/badge/Swift-2.3-orange.svg)
+[![Language](https://img.shields.io/badge/Swift-3.0-orange.svg)](https://swift.org)
```

Comment:
For accessibility reasons (= because people with visual deficiency use the alt description of an image to help them understand what the image is) I think it would be better for non-automatic badges like this one (unlike the one whose image is auto-generated) to have a descriptive text, like `Language: Swift 3.0` instead of just "Language"?